atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/LockWatchValueScopingCacheImpl.java

```diff
@@ -256,7 +267,7 @@ private Optional<LockWatchVersion> computeMaxUpdateVersion(TransactionsLockWatch
     }
 
     private synchronized boolean shouldUpdateVersion(LockWatchVersion updateVersion) {
-        return currentVersion.isEmpty() || currentVersion.get().version() < updateVersion.version();
+        return currentVersion.isEmpty() || currentVersion.orElseThrow().version() < updateVersion.version();
```

Comment:
I had no idea that `orElseThrow()` did exactly the same as `get()` under the hood!

Reply:
yep, this just cleans up an intellij warning and makes it clear we expect value present